iphone/cli/commands/_build.js

```diff
@@ -116,7 +116,7 @@ function iOSBuilder() {
 
 	// when true, uses the JavaScriptCore that ships with iOS instead of the original Titanium version
 	this.useJSCore = false;
-
+	this.runOnMainThread = true;
```

Comment:
Add whitespace before and after this line. Put a comment on the line above it to describe what this flag does.

Should this default to `true`?